account/models.py

```diff
@@ -8,3 +9,17 @@ class Account(models.Model):
 
     class Meta:
         db_table = 'accounts'
+
+class Product_wishlist(models.Model):
```

Comment:
클래스 이름은 카멜케이스로 사용합니다.
```suggestion
class ProductWishlist(models.Model):
```